src/primitives/block.cpp

```diff
@@ -15,7 +15,7 @@ uint256 CBlockHeader::GetHash() const
     return SerializeHash(*this);
 }
 
-uint256 CBlock::BuildMerkleTree(bool* fMutated) const
+uint256 CBlock::ComputeMerkleRoot(bool* fMutated) const
```

Comment:
I'm not sure this is so much "preferred", it's how it happens to be written between two more-or-less equivalent ways to formulate it.